boot/seal.go

```diff
@@ -107,10 +107,10 @@ func sealKeyToModeenv(key secboot.EncryptionKey, model *asserts.Model, modeenv *
 		}
 	}
 	sealKeyParams := &secboot.SealKeyParams{
-		ModelParams:             modelParams,
-		KeyFile:                 filepath.Join(InitramfsEncryptionKeyDir, "ubuntu-data.sealed-key"),
-		TPMPolicyUpdateDataFile: filepath.Join(InstallHostFDEDataDir, "policy-update-data"),
```

Comment:
Just for my own understanding:
We no longer update the "policy-update-data" key filename and it's now "tpm-policy-auth-key" ?
This means this change will break upgrades (again fine, just double checking)?

Reply:
I think the new file name is more appropriate, but indeed this change will break old installs.

Reply:
Yeah, I like the new name better too